staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
should we name the file legacy_types.go as well?

Reply:
I feel we should first make new Scheduler CC stable - which have policy structs inline - and then mark it as legacy_types.go. (and also mark it as deprecated)